frontend/src/lib/k8s/api/v2/useKubeObjectList.ts

```diff
@@ -313,17 +313,29 @@ export function useKubeObjectList<K extends KubeObject>({
     queryParams: cleanedUpQueryParams,
   });
 
+  const sortedItems = useMemo(() => {
```

Comment:
@sniok , I wonder if it's a good idea to special case Namespace here, instead of just where it's needed (UI) or overriding useList in KubeNamespace.

Reply:
it's also in the list on the namespace page but since there is no need beyond the UI for sorting, adjustments to the UI code should be fine